src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,124 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    public function getBundleData()
+    {
+        $bundles = array();
+        $installedPackages = $this->getInstalledPackages();
+        $enabledBundles = $this->getEnabledBundles();
+
+        foreach ($this->getEnabledBundles() as $bundleName => $bundlePath) {
+            foreach ($installedPackages as $packageName => $packageVersion) {
+                if (strpos($bundlePath.'/', '/vendor/'.$packageName.'/')) {
```

Comment:
You assume here that people install their dependencies in the `<root>/vendor` folder, but sometimes this is not the case (dependencies folder can be changed in `composer.json`) .